api/v1beta1/cinder_types.go

```diff
@@ -90,3 +163,16 @@ type CinderList struct {
 func init() {
 	SchemeBuilder.Register(&Cinder{}, &CinderList{})
 }
+
+// IsReady - returns true if service is ready to serve requests
+func (instance Cinder) IsReady() bool {
+	ready := (instance.Spec.CinderAPI.Replicas < 1 || instance.Status.CinderAPIReadyCount > 0) &&
```

Comment:
I'll defer to your understanding of the intended business logic, as I was just essentially using the same logic we had in OSP-D operator for `OSBaremetalSet`, where you could have an "empty" CR (0 replicas/BMHs requested) and the set would be considered "ready".  But since that doesn't make sense here, I will fix this

Reply:
In OSP-D it makes sense if it is an empty role and in the tripleo environment file the role was set to 0 count. here I think if the spec has replica > 0, the status also needs a > 0 ready count.